.gitignore

```diff
@@ -1,4 +1,5 @@
 .vscode
+.idea
```

Comment:
```suggestion
```

Let's keep IDE-specific lines out of the codebase. You can ignore these locally in `$XDG_CONFIG_HOME/git/ignore` or `$GIT_DIR/info/exclude`.